feature-utils/poly-look/test/react-components/overlays/base.test.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import BaseOverlay from "../../../src/react-components/overlays/base";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom";
+
+const testClassName = "test1";
+const testId = "testId";
+const testComponent = (
+  <BaseOverlay className={testClassName}>
+    <div data-testid={testId}></div>
+  </BaseOverlay>
+);
+
+const centeredTestComponent = (
+  <BaseOverlay className={testClassName} centered={true}></BaseOverlay>
+);
+
+it("renders correctly", () => {
+  const { container, getByTestId } = render(testComponent);
+  expect(container).toBeTruthy();
+  const baseOverlay = getByTestId("base-overlay-test");
```

Comment:
Would it be problematic if you define the `testId` here and not where the class is defined? I don't see any real need to do it there, since we will actually be using it here only.
Also, there's the problem of the actual testID, which is defined by two literals, one here and the other there. Difficult for refactoring.

Reply:
Hmm I don't fully get this. Make baseOverlay have a testId prop? That seems like something we wouldn't want to do

Reply:
I was looking at `BaseOverlay` and thought that maybe you could add a data-testId like this:

```jsx
<BaseOverlay className={testClassName} data-testId='whatever'>
```

But looking at its definition I see you don't have variable props, so this is not possible, I guess. That would save us including content that's just for testing in the base class. But that's probably not the case, so never mind.